akash1/HTML/ASSIn4/tableQ1.html

```diff
@@ -1,50 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>tableQ1</title>
     <style>
-        table,
-        th,
-        td {
-            border-collapse: collapse;
-            padding: 20px 20px 20px 20px;
-            border: 3px solid black;
-        }
-        table,
-        caption,
-        td,
-        th {
-            background-color: gray;
-        }
+      table,
+      th,
+      td {
```

Comment:
add table, td, th in single line. like
table, th, td{ }...

Reply:
okey..after making indentation it gets this format.